templates/kapeta/resource-type-rest-client/mocks.handlers.ts.hbs

```diff
@@ -1,7 +1,4 @@
-//#FILENAME:src/mocks/handlers.ts:write-always
-//
-// GENERATED SOURCE - DO NOT EDIT
-//
+//#FILENAME:src/mocks/handlers.ts:create-only
```

Comment:
The `handlers.ts` file should not be edited. It should be regenerated whenever the API changes